### ISSUE TITLE
No coverage reported for PHPStorm 2019.2.5

### ISSUE THREAD

**Issue by remorhaz** [sebastianbergmann/php-code-coverage#710]
| Q                         | A
| --------------------------| ---------------
| php-code-coverage version | 7.0.9
| PHP version               | 7.3.11
| Driver                    | Xdebug
| Xdebug version (if used)  | 2.8.0
| Installation Method       | Composer
| Usage Method              | PHPUnit
| PHPUnit version (if used) | 8.4.3

After updating from 7.0.8 to 7.0.9 PHPStorm 2109.2.5 stops showing the coverage when running tests from IDE. No errors in IDE logs, it just behaves like all the code is excluded from coverage. Downgrading to 7.0.8 solves the problem, so the cause is probably in last php-code-coverage release.

`phpunit.xml`:
```
<?xml version="1.0" encoding="UTF-8" ?>
<phpunit
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/8.4/phpunit.xsd"
        forceCoversAnnotation="true"
        colors="true"
        defaultTestSuite="unit">
    <testsuites>
        <testsuite name="unit">
            <directory>tests/</directory>
        </testsuite>
    </testsuites>
    <filter>
        <whitelist
                processUncoveredFilesFromWhitelist="true">
            <directory suffix=".php">src/</directory>
        </whitelist>
    </filter>
</phpunit>
```

**Comment by sebastianbergmann** [comment]
While I use PhpStorm, I do not use its UI to run tests or look at coverage. Therefore I cannot reproduce the issue.

Can you narrow down which of the [commits](https://github.com/sebastianbergmann/php-code-coverage/compare/7.0.8...7.0.9) made between 7.0.8 and 7.0.9 is responsible for this?

**Comment by sebastianbergmann** [comment]
The only thing that would make sense to be the cause for this is #705 (and #706). CC @danielpalme

**Comment by remorhaz** [comment]
[This is the change](https://github.com/sebastianbergmann/php-code-coverage/compare/7.0.8...7.0.9#diff-19b24179073358fac0d4c1ab22d06243R53) that brought the problem. Rolling back to `$xmlFile->setAttribute('name', $item->getPath());` in `Report/Clover.php` solves the issue.

**Comment by sebastianbergmann** [comment]
I have reverted #705 and #706 and have released 7.0.10. Thanks!
